sql/StoredProcedures/SelectionStoredProcedures.sql

```diff
@@ -0,0 +1,44 @@
+DROP procedure IF EXISTS `search_for_company`;
+DELIMITER $$
+CREATE PROCEDURE `search_for_company`(name_search varchar(50) )
+  BEGIN
+    SELECT * FROM companies WHERE name = name_search;
+  END$$
+DELIMITER ;
+
+
+DROP procedure IF EXISTS `search_for_job_by_title`;
+DELIMITER $$
+CREATE PROCEDURE `search_for_job_by_title`( title_search varchar(50))
+  BEGIN
+    SELECT * FROM jobs WHERE title = title_search;
+  END$$
+DELIMITER ;
+
+
+
+DROP procedure IF EXISTS `search_for_user_by_skill`;
+DELIMITER $$
+CREATE DEFINER=`root`@`localhost` PROCEDURE `search_for_user_by_skill`(skill_id_in int)
```

Comment:
Can you please also remove the definer from here? :angel:

Reply:
Done